### ISSUE TITLE
Can Chinese labels be supported when exporting

### ISSUE THREAD

**Issue by zwd22** [mifi/lossless-cut#2173]
### The fewer issues I have to read, the more new features I will have time to implement, so I ask that you please try these things first

- [X] Check out all the menus and keyboard shortcuts. Maybe the feature already exists?
- [X] Try with the [newest version from GitHub](https://github.com/mifi/lossless-cut/releases/latest)
- [X] Read the [documentation](https://github.com/mifi/lossless-cut) and [FAQ, Known issues, Troubleshooting](https://github.com/mifi/lossless-cut/blob/master/issues.md) to see if the feature already exists
- [X] See if someone already suggested this under [Issues](https://github.com/mifi/lossless-cut/issues) or [Discussions](https://github.com/mifi/lossless-cut/discussions)

### Description

Everything is very good with this software, that is, when exporting English labels can be exported normally, but Chinese labels are spaces after exporting, I really need this function

**Comment by mifi** [comment]
chinese characters are already supported. See https://github.com/mifi/lossless-cut/blob/master/issues.md#output-file-name-is-missing-characters

**Comment by zwd22** [comment]
> chinese characters are already supported. See https://github.com/mifi/lossless-cut/blob/master/issues.md#output-file-name-is-missing-characters已经支持汉字了。请参阅https://github.com/mifi/lossless-cut/blob/master/issues.md#output-file-name-is-missing-characters

Indeed, that's great, thank you very much, just need to turn off the “Output file names”!
![LosslessCut_gtbgYC8JOp](https://github.com/user-attachments/assets/f1d24a07-6ca9-4b9a-93bf-d3f0e0e2fb73)
